app/models/user.rb

```diff
@@ -82,7 +82,11 @@ def client_admin?
   end
 
   def admin?
-    self.has_role?('admin')
+    has_role?('admin')
+  end
+
+  def not_admin?
+    !admin?
   end
```

Comment:
I don't feel super strongly about this method in particular, but the extension of this argument is that you then need two methods for every method that returns a boolean, which seems excessive.